vnext/ReactUWP/Views/Image/ImageViewManager.cpp

```diff
@@ -33,6 +33,8 @@ struct json_type_traits<react::uwp::ImageSource>
         source.uri = item.second.asString();
       else if (item.first == "method")
         source.method = item.second.asString();
+      else if (item.first == "bundleRootPath")
+        source.bundleRootPath = item.second.asString();
```

Comment:
This is not necessary as we're getting the bundle root path from calling GetBundleRootPath and not from data.

Reply:
Right, did not see where this was being called, sorry. Removing.